examples/example.js

```diff
@@ -22,6 +22,15 @@ fastify
   .post('/', schema, function (req, reply) {
     reply(null, { hello: 'world' })
   })
+  .head('/', schema, function (req, reply) {
+    reply(null, { hello: 'world' })
```

Comment:
This example is not completely correct, head should not return a body, is just to get the headers without download the body too :)